source/topics/account/how-to-invite-training-user.md

```diff
@@ -0,0 +1,13 @@
+To invite new team members to training: 
+
+1) Create a _Training-only_ Role (if it does not yet exist)
+
+- Dashboard => Settings => Roles => `Create role` => _No Permissions_
```

Comment:
I would use prose (e.g., [Papertrail article](https://github.com/aptible/support/blob/master/source/topics/paas/how-to-set-up-papertrail.md)). Also, code formatting (backticks) should not be used here. Finally, Markdown ordered lists should be `1.`, etc. instead of `1)`.